### ISSUE TITLE
Infinite loop of prompts to install update between 14.5.1 and 14.6.13.

### ISSUE THREAD

**Issue by ThomasOwens** [jgraph/drawio-desktop#582]
### Preflight Checklist
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
When opening the desktop app and downloading the update, the app continues to prompt for an update after installing the update. This is the same issue as described in #518 except with the 14.6.13 release.

**To Reproduce**
The reproduction steps are the same as #518.

**Expected behavior**
After installing the update, the update should be installed and I should not be reprompted to install an update when the app reopens.

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 14.5.1

**Desktop (please complete the following information):**
 - OS: Windows

**Additional context**
I resolved the problem by manually downloading and installing the update from the GitHub release page. If the auto-updater is an issue, perhaps it would be a good idea to remove that functionality. Instead of downloading and installing the update, allowing the user to have a link to the GitHub release page to download and install the right version would be more consistent.

**Comment by raindropsfromsky** [comment]
I too faced this, on Windows 10 Home, 64 bits.

Finally I used the Ccleaner application to uninstall Draw. At that time, I found that both 14.5.1 and 14.6.13 are there (the older version should have been uninstalled).

Further, I was able to uninstall 14.5.1, but the 14.6.13 version could not be uninstalled at all.
Finally, I deleted it and restarted the laptop.

Then I downloaded 14.6.13 from GitHub afresh, and did a clean installation.
Now the endless loop has stopped.
